lib/cli/src/commands/app/create.rs

```diff
@@ -1,34 +1,54 @@
 //! Create a new Edge app.
 
-use std::path::PathBuf;
-
-use anyhow::{bail, Context};
+use crate::{
+    commands::AsyncCliCommand,
+    opts::{ApiOpts, ItemFormatOpts},
+    utils::{
+        load_package_manifest,
+        package_wizard::{CreateMode, PackageType, PackageWizard},
+    },
+};
+use anyhow::Context;
 use colored::Colorize;
-use dialoguer::Confirm;
-use edge_schema::schema::StringWebcIdent;
+use dialoguer::{Confirm, Select};
 use is_terminal::IsTerminal;
-use wasmer_api::{
-    types::{DeployAppVersion, Package, UserWithNamespaces},
-    WasmerClient,
+use std::{collections::HashMap, path::PathBuf, str::FromStr};
+use wasmer_api::{types::UserWithNamespaces, WasmerClient};
+use wasmer_config::{
+    app::AppConfigV1,
+    package::{NamedPackageIdent, PackageSource, Tag},
 };
 
-use crate::{
-    commands::{
-        app::{deploy_app_verbose, AppConfigV1, DeployAppOpts, WaitMode},
-        AsyncCliCommand,
-    },
-    opts::{ApiOpts, ItemFormatOpts},
-    utils::package_wizard::{CreateMode, PackageType, PackageWizard},
-};
+use super::deploy::CmdAppDeploy;
+
+async fn write_app_config(app_config: &AppConfigV1, dir: Option<PathBuf>) -> anyhow::Result<()> {
+    let raw_app_config = app_config.clone().to_yaml()?;
+
+    let app_dir = match dir {
```

Comment:
very tiny nitpick: seeems to me that the caller should always pass in a valid path here, and should be resposible for falling back to current dir

Reply:
@xdoardo

Reply:
https://github.com/wasmerio/wasmer/issues/4598